spec/lib/api/v3/work_packages/work_package_representer_spec.rb

```diff
@@ -734,6 +738,42 @@
             it_behaves_like 'has no link' do
               let(:link) { 'parent' }
             end
+
+            it 'renders empty ancestors' do
+              expect(subject).to have_json_size(0).at_path('_links/ancestors')
+            end
+          end
+        end
+
+        context 'ancestors' do
+          let(:root) { FactoryGirl.create(:work_package, project: project) }
+          let(:work_package) do
+            FactoryGirl.create(:work_package, parent: intermediate, project: project)
+          end
+
+          context 'when intermediate is visible' do
+            let(:intermediate) do
+              FactoryGirl.create(:work_package, parent: root, project: project)
+            end
```

Comment:
I think it would be better to simply stub the `visible_ancestors` method here for speed. We should strive to get the representer specs to be fast which will not be possible if we access the DB in them.